internal/util/get/get.go

```diff
@@ -257,22 +257,15 @@ func (c *Command) DefaultValues() error {
 // cloneFrom values.
 func (c *Command) upsertKptfile(spec *git.RepoSpec) error {
 	// read KptFile cloned with the package if it exists
-	kpgfile, err := kptfileutil.ReadFile(c.Destination)
+	var contents []byte
+	contents, err := ioutil.ReadFile(filepath.Join(c.Destination, kptfile.KptFileName))
```

Comment:
Why does each subcommand need to do these low-level operations such as reading a file and marshalling it? We should extract this logic and reuse it across subcommands.